TripKit/Classes/helpers/TKAccessibilityHelper.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TKAccessibilityHelper.swift
+//  TripKit-iOS
+//
+//  Created by Kuan Lun Huang on 22/2/18.
+//  Copyright © 2018 SkedGo. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+/// This class provides information about what accessibility features we support.
+/// in a given region. It currently only supports wheelchair. You can ask whether
+/// wheelchair information is available, for instance.
+@objc public class TKAccessibilityHelper: NSObject {
+  
+  let region: SVKRegion
+  
+  @objc public var isWheelchairInfoAvailable = false {
+    didSet {
+      rx_wheelchairInformationUpdated.onNext(())
+    }
+  }
+  
+  public let rx_wheelchairInformationUpdated = PublishSubject<Void>()
+  
+  private let disposeBag = DisposeBag()
+  
+  // MARK: -
+  
+  @objc public init(region: SVKRegion) {
+    self.region = region
+    super.init()
+  }
+  
+  @objc public func fetchWheelchairSupportInformation() {
```

Comment:
Please change this to have a completion handler. Otherwise, when using this from Objective C you'd never know when the data is available. This is problematic as it starts out as false rather than in an unknown state - so if you call later if the wheelchair is supported and it returns false, does it mean that it's not supported and just that the server didn't return yet?

Reply:
Also can we get rid off everything else this and only have this as a single static method with a completion block? Worry about the caller caching this.

Reply:
Even better: If it's just a single method to fetch accessibility/wheelchair info, it's better to just make it one of the methods in `TKBuzzInfoProvider` where we keep all these methods for fetching info about a region. Could give that class a better name though, like `TKRegionInfoProvider`.

Reply:
Done